### ISSUE TITLE
Set up `ResourceCard` component

### ISSUE THREAD

**Issue by lpatmo** [codebuddies/frontend#18]
We need a `ResourceCard` component that uses material-ui's `Card` component and looks like this:

![image](https://user-images.githubusercontent.com/4512699/67188732-cf9d7d80-f3a1-11e9-8dfc-8a1653d008e7.png)

Notes:
- See https://material-ui.com/components/cards/ for reference
- Until #15 is tackled, we can pass in fixed example data as props. 
- Let's use material-ui's `icons` library for the icon. It should be a `bookmark` icon instead of a `heart` icon.

**Comment by agarwalamn** [comment]
Hey! 
I have a query, do you want all these things in different files? 
Because in your repo there is only plain react initial code

**Comment by lpatmo** [comment]
@agarwalamn Ah sorry, I have some initial setup in https://github.com/codebuddies/react-concept/pull/10, and once that is merged we can add `ResourceCard` into the components folder!

**Comment by lpatmo** [comment]
Update: it's been merged! I'm going to make another PR to return some sample data via `axios`, and then we should be able to move forward.

**Comment by lpatmo** [comment]
@agarwalamn Let me know if you're still interested in working on this issue! I merged in some initial setup, so if you pull the latest you should see this: https://cb-react-concept.netlify.com (click on "Resources" on the live site; there is an issue with going to https://cb-react-concept.netlify.com/resources that will be fixed soon)

I don't think we should make those cards look *exactly* like the mockup anymore, but it would be good to get these additional details displayed with `material-ui`:

1/ Tags
2/ Learning notes
3/ Hangouts proposed / Hangouts completed
4/ Questions / # resolved

**Comment by agarwalamn** [comment]
@lpatmo I will!

**Comment by rozenraymond** [comment]
I'm happy to work on this too if no one else pick up :)

**Comment by lpatmo** [comment]
> I don't think we should make those cards look exactly like the mockup anymore, but it would be good to get these additional details displayed with material-ui:

1/ Tags
2/ Learning notes
3/ Hangouts proposed / Hangouts completed
4/ Questions / # resolved

-----
The above piece still needs to be tackled. Might do this myself if no one else (or @rozenraymond) claims it.

**Comment by rozenraymond** [comment]
happy to claims this issue 😃

**Comment by lpatmo** [comment]
Closing this issue, since not everyone is aligned on the final design for the resource card.
